actpool/actpool.go

```diff
@@ -408,7 +422,8 @@ func (ap *actPool) removeConfirmedActs() {
 		// Remove all actions that are committed to new block
 		acts := queue.FilterNonce(pendingNonce)
 		ap.removeInvalidActs(acts)
-
+		//del actions in destination map
+		ap.deleteAction(acts...)
```

Comment:
Don't you delete actions already in removeInvalidActs?

Reply:
I think it's invalid action,need to delete confirmed actions